### PR TITLE
Add targets to manage Redis

### PR DESCRIPTION
**Pull request by jeckersb**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,6 +37,11 @@ MEMCACHED           ?= config/samples/memcached_v1beta1_memcached.yaml
 MEMCACHED_CR        ?= ${OPERATOR_BASE_DIR}/infra-operator/${MEMCACHED}
 MEMCACHED_IMG       ?= ${SERVICE_REGISTRY}/${SERVICE_ORG}/openstack-memcached:current-tripleo
 
+# Redis
+REDIS               ?= config/samples/redis_v1beta1_redis.yaml
+REDIS_CR            ?= ${OPERATOR_BASE_DIR}/infra-operator/${REDIS}
+REDIS_IMG           ?= registry.redhat.io/rhel9/redis-6:latest
```
[the remark this answers is not in the document]
Comment (jeckersb) [reply Makefile]: Fixed, sorry for the delay I just realized I was misfiltering github emails!

**Comment by beagles** [comment]
Just a heads up that a redis target that uses a redis specific checkout and CR path to avoid interfering with other infra operator based services has been merged. See PR 748 and 762

**Comment by stuggi** [comment]
closing since redis was removed in https://github.com/openstack-k8s-operators/install_yamls/pull/801

**Pull request closed without merging**